pkg/aws/eni/routing/routing.go

```
@@ -119,6 +125,12 @@ func (info *RoutingInfo) Configure(ip net.IP, mtu int, masq bool) error {
 // with this ID, and because this table ID equals the ENI ifindex, it's stable
 // to rely on and therefore can be reused.
 func Delete(ip net.IP) error {
+	if ip.To4() == nil {
+		log.WithFields(logrus.Fields{
+			"IP": ip,
+		}).Warning("Unable to Delete, the endpoint address is not an IPv4 address in ENI mode")
+		return nil
```

We want to return an error here as well, same as other comment.